internal/venice-common/src/main/resources/avro/KafkaMessageEnvelope/v11/KafkaMessageEnvelope.avsc

```diff
@@ -209,6 +209,28 @@
                       "type": "boolean"
                     }
                   ]
+                }, {
+                  "name": "StartOfBufferReplay",
```

Comment:
Just curious - isn't this only used by OBO model? Since they are all gone, why is it still a concern here?